src/modeler/optimConfig/include/antares/modeler/optimConfig/optimConfig.h

```diff
@@ -32,6 +34,8 @@ enum class Location
     SUBPROBLEMS
 };
 
+std::ostream& operator<<(std::ostream& os, Location loc);
```

Comment:
If this code is here because of tests, it shouldn't be. 
Code for tests should not colonize production code (as you already know).